Makefile

```diff
@@ -376,10 +376,7 @@ setup-opa-policies:
 		echo "Downloading policies from Forklift GitHub repository..."; \
 		mkdir -p $(FORKLIFT_POLICIES_TMP_DIR); \
 		curl -L https://github.com/kubev2v/forklift/archive/main.tar.gz \
-			| tar -xz -C $(FORKLIFT_POLICIES_TMP_DIR) \
-				--wildcards '*/validation/policies/io/konveyor/forklift/vmware/*' \
-				--strip-components=1 \
-				--exclude='*/..*' --exclude='*/.*'; \
+			| tar -xz -C $(FORKLIFT_POLICIES_TMP_DIR) --strip-components=1; \
```

Comment:
I don't really know why its needed, but why did you remove the "--wildcard" and "--exclude" flags?

Reply:
So as far as I understood (I used CursorAI), these 2 flags "--wildcard" and "--exclude" are used for the GNU version of tar command which is the standard on most Linux systems, but macOS uses a different version of tar. so what was changed is the way we are using tar - to be more generic and universal. I did not tested it on fedora/rhel machines thought